python/llm/src/ipex_llm/transformers/models/phi3.py

```diff
@@ -179,8 +189,12 @@ def model_forward(
     ):
         # IPEX-LLM OPT: kv cache but no sdp (its head_dim 96, cannot use sdp)
         use_cache = use_cache if use_cache is not None else self.config.use_cache
+        use_quantize_kv = (use_quantize_kv_cache(self.layers[0].mlp.down_proj, input_ids) and
+                           self.config.hidden_size // self.config.num_attention_heads in [64, 128])
```

Comment:
maybe put `self.config.hidden_size // self.config.num_attention_heads in [64, 128` this check into `use_quantize_kv_cache`  ？

Reply:
yes, we'll move it later